back-end/src/sciler/config/checkConfig.go

```diff
@@ -405,7 +405,7 @@ func checkConstraintsDeviceStringType(errorParameters []string, constraint Const
 		return []string{fmt.Sprintf("level III - implementation error: on rule %s, constraint on device with id %s: comparison '%s' is not valid",
 			errorParameters[0], errorParameters[1], comparison)}
 	}
-	if comparison != "eq" && comparison != "not" {
+	if comparison != "eq" && comparison != "not" && comparison != "eqwt" && comparison != "not eqwt" {
```

Comment:
the current `checkConstraintsDeviceNumericType` now also allows for `eqwt` and `not eqwt`, so that should be changed

Reply:
done